### ISSUE TITLE
If Tauon is maximised, and the Ubuntu taskbar is auto-hidden, there's no way to make it visible

### ISSUE THREAD

**Issue by Darkmaster006** [Taiko2k/Tauon#519]
I have my Ubuntu taskbar on auto-hide mode, so I can only make it visible if I hover over the botton (where the taskbar is) with my mouse, or show it with Super+I. Now, if the player is in "fullscreen" mode, the taskbar, when hovered over, gets buggy; when made visible with Super+I, does not show (if I hit Super+I to hide it, it shows for a flicking second). Basically, it's like it tries to show it but it hides it immediately, in both cases. If a video example is needed, I am happy to provide it. I think we can call this a bug since this has not happened to me with any other program.

Ah, I also found another minor bug, when using the 'window panel to the left' that you get with 'Alt + Space' that shows you 'Minimise/Maximise, Unmaximise, Move, Resize, etc' if you press Space or Enter to select an option, it also presses Space or Enter in the player, hence hitting either a song (so that it is played again from the beginning) or an album if you've used tab to use the 'keyboard control mode. This is also the case with the shortcuts 'Alt + F7' and 'Alt + F8' which Move and Resize a window respectively. When using these shortcuts, you use 'Esc' if you don't want to save the changes, and you use either 'Spacebar' or 'Enter' to save them. 'Enter', in this case, plays the song again from the beginning. 'Space', of course, plays or stops the song being played (or plays a song from the beginning if none is playing, I guess).

Hope this is helpful, and sorry for opening so many issues at once haha. I am running Ubuntu 20.04 with the Dash to Panel extension, if that helps.

**Comment by Taiko2k** [comment]
To clarify do you mean fullscreen or maximised? In the former not being able to access the taskbar would be expected. Either way I don't think there's much I can do about bugs related to this.

As for the keypress issue, this is a common issue but I don't really understand how to handle it, surely the WM should take away key input if its not supposed to go to that window 😕

**Comment by Darkmaster006** [comment]
> To clarify do you mean fullscreen or maximised? In the former not being able to access the taskbar would be expected. Either way I don't think there's much I can do about bugs related to this.
> 
> As for the keypress issue, this is a common issue but I don't really understand how to handle it, surely the WM should take away key input if its not supposed to go to that window confused

I feel like Fullscreen (F11) and Maximise do the same in Tauon. Either way, in both, the taskbar does not work. It is not really a problem for me (since I use Alt + Tab) but just thought the bug should be reported.

And... yeah, I don't get why the WM does not take away the key input when, well, it does in all other applications.

**Comment by jansol** [comment]
Hmm, when pressing F11 to make tauon fullscreen the minimize and close buttons in the top right corner still work normally, but the maximize button does not seem to do anything. In a full screen mode I'd expect all three buttons to disappear as part of window decorations.

When maximizing tauon using the maximize button that button restores the previous window size as expected. Doing the same in the F11 fullscreen mode would make sense since there is no visual indicator of which one tauon is in. This is also how "modern" fullscreen on macOS behaves (title bar is revealed on hover and maximize/restore returns to normal-sized window).

**Comment by Taiko2k** [comment]
@jansol Yeah it makes sense that the maximize button should un-fullscreen instead of doing nothing when in fullscreen. Ive committed the change now.

**Comment by C0rn3j** [comment]
What's left of the remaining issues nowadays, if any?

**Comment by C0rn3j** [comment]
If there are indeed any issues left in current SW versions, please open a new issue for them.
